main.tf.json

```diff
@@ -201,6 +201,59 @@
                 }
             }
         },
+        {
+            "aws_iam_policy_document": {
+                "clumio_dynamodb_policy_document": {
```

Comment:
Just curious why this one is not protected w/ the `is_dynamodb_enabled` variable?

Reply:
Seems like many of the newly added policies are not protected w/ the variable?

Reply:
Yes. New resources should be protected with the appropriate flag. Will make the change.

Reply:
@lawrence-clumio these are data_source statements. We don't protect the data_sourceAll the resources are protected appropriately

Reply:
Not needed per our conversation.